### DISCUSSION TITLE
Connecting to sharepoint

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#6153] [Q&A] [answered]
Does anyone know if it is possible for Mirth to connect to Sharepoint and pick up a file? If so, how would you do that?

**Comment by pacmano1** [comment, marked as answer]
Sharepoint has a REST API, pretty sure it has for a very long time.

https://learn.microsoft.com/en-us/sharepoint/dev/sp-add-ins/get-to-know-the-sharepoint-rest-service?tabs=csom
